packages/odyssey-react/src/components/Tooltip/Tooltip.module.scss

```diff
@@ -40,11 +41,11 @@
   }
 
   &::after {
-    border-width: calc(#{$spacing-xs} - 2px);
+    border-width: calc(var(--TailWidth) - calc(var(--TipBorderWidth) * 2));
```

Comment:
This formula won't work perfectly with all values, but should allow basic theming of the tail and tip size.